test/compression.js

```diff
@@ -465,6 +475,22 @@ describe('compression()', function () {
     })
   })
 
+  if (hasBrotliSupport) {
```

Comment:
Instead of excluding the test, have the test always be there but show up as skipped in the mocha output when the support is missing.